examples/chain_client/offchain_vaults/6_RedeemDerivativeVault.py

```diff
@@ -0,0 +1,113 @@
+import asyncio
+import time
+
+from pyinjective.async_client import AsyncClient
+from pyinjective.transaction import Transaction
+from pyinjective.core.network import Network
+from pyinjective.wallet import PrivateKey
+
+def mint_amount_from_response(resp):
+    for ev in resp.tx_response.events:
+        if ev.type != "wasm-lp_balance_changed":
+            continue
+        for attr in ev.attributes:
+            if attr.key == "mint_amount":
+                return attr.value
+    return ""
```

Comment:
What's the return type when returning attr.value? If this is always a string, then I am fine with returning "" instead of None. Otherwise I would try to return a None-like object in the same type